src/runtime/methodbinder.cs

```diff
@@ -180,6 +185,11 @@ internal MethodBase[] GetMethods()
         /// </remarks>
         internal static int GetPrecedence(MethodBase mi)
         {
+            if (mi == null)
+            {
+                return -1;
+            }
+
```

Comment:
I think `int.Max` would be more appropriate. Higher the number - lower the priority.

UPD. can this even be null?